src/main/java/com/alkemy/ong/domain/slides/SlidesService.java

```diff
@@ -11,5 +11,7 @@ public SlidesService(SlidesGateway slidesGateways){
         this.slidesGateways = slidesGateways;
     }
 
+    public void delete(Long id){ slidesGateways.delete(id); }
```

Comment:
que pasa si quiero borrar un id que no existe? Debería devolver un 404 resource not found. 
Podes crear una excepción ResourceNotFoundException y dsp en el global exception handler mapear que cuando lleguen este tipo de excepciones devuelvas el status code 404. 
Este delete respeta el borrado lógico verdad?

Reply:
Listo el pollo.

- El PostMan cuando lo encuentra y lo borra:
![imagen](https://user-images.githubusercontent.com/52411075/155356957-47ad6ac2-42c5-47bb-a03e-edb974f606bc.png)
_Solo devuelve un codigo 204_

- El PostMan cuando no encuentra: 
![imagen](https://user-images.githubusercontent.com/52411075/155356623-55f6073c-1ebb-4db2-9f7d-53c59d8108d3.png)

- MySQL con el softDelete
![imagen](https://user-images.githubusercontent.com/52411075/155357167-74352bb7-fe95-4c12-921c-be0a7865ef59.png)